<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Static5</class>
 <widget class="QWidget" name="Static5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gen 5 Static</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="title">
      <string>Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileTIDValue">
        <property name="text">
         <string notr="true">12345</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelProfileMACAddress">
        <property name="text">
         <string>MAC Address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileMACAddressValue">
        <property name="text">
         <string notr="true">9BF123456</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7" rowspan="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="labelProfileVCount">
        <property name="text">
         <string>VCount</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="labelProfileVCountValue">
        <property name="text">
         <string notr="true">2F</string>
        </property>
       </widget>
      </item>
      <item row="0" column="10" rowspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QLabel" name="labelProfileGxStat">
        <property name="text">
         <string>GxStat</string>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QLabel" name="labelProfileGxStatValue">
        <property name="text">
         <string notr="true">6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="13" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="14">
       <widget class="QLabel" name="labelProfileKeypresses">
        <property name="text">
         <string>Keypresses</string>
        </property>
       </widget>
      </item>
      <item row="0" column="15">
       <widget class="QLabel" name="labelProfileKeypressesValue">
        <property name="text">
         <string>None, 1, 2, 3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileSIDValue">
        <property name="text">
         <string notr="true">54321</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="labelProfileDSType">
        <property name="text">
         <string>DS Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="labelProfileDSTypeValue">
        <property name="text">
         <string>DS Original/Lite</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QLabel" name="labelProfileTimer0">
        <property name="text">
         <string>Timer0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QLabel" name="labelProfileTimer0Value">
        <property name="text">
         <string notr="true">621-621</string>
        </property>
       </widget>
      </item>
      <item row="1" column="11">
       <widget class="QLabel" name="labelProfileVFrame">
        <property name="text">
         <string>VFrame</string>
        </property>
       </widget>
      </item>
      <item row="1" column="12">
       <widget class="QLabel" name="labelProfileVFrameValue">
        <property name="text">
         <string notr="true">5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="14">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="1" column="15">
       <widget class="QLabel" name="labelProfileGameValue">
        <property name="text">
         <string>White</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabRNGSelector">
     <widget class="QWidget" name="tabStatic">
      <attribute name="title">
       <string>Generator</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxGeneratorRNGInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButtonGeneratorLead">
            <property name="text">
             <string>Synchronize</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelGeneratorMaxAdvances">
            <property name="text">
             <string>Max Advances</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="TextBox" name="textBoxGeneratorSeed"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGeneratorEncounter">
            <property name="text">
             <string>Encounter</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorEncounter">
            <item>
             <property name="text">
              <string>Static</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Roamer</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelGeneratorSeed">
            <property name="text">
             <string>Seed</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBoxGeneratorLead"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelGeneratorInitialAdvances">
            <property name="text">
             <string>Initial Advances</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="TextBox" name="textBoxGeneratorInitialAdvances">
            <property name="text">
             <string notr="true">0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="TextBox" name="textBoxGeneratorMaxAdvances">
            <property name="text">
             <string notr="true">100000</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelGeneratorMethod">
            <property name="text">
             <string>Method</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorMethod">
            <item>
             <property name="text">
              <string>IVs</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IVs C-Gear</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PIDRNG</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButtonGenerate">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Generate</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCalculateInitialAdvances">
              <property name="text">
               <string>Calculate Initial Advance</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxGeneratorFilters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" rowspan="2">
           <widget class="Filter" name="filterGenerator" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="TableView" name="tableViewGenerator">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSpreadSearcher">
      <attribute name="title">
       <string>Searcher</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxSearcherFilters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0" rowspan="2">
           <widget class="Filter" name="filterSearcher" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="TableView" name="tableViewSearcher">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxSearcherRNGInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="1">
           <widget class="QLabel" name="labelSearcherLead">
            <property name="text">
             <string>Lead</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="labelSearcherStartDate">
            <property name="text">
             <string>Start Date</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelSearcherMethod">
            <property name="text">
             <string>Method</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="5">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButtonSearch">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Search</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCancel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="1" colspan="5">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="labelSearcherMinAdvance">
              <property name="text">
               <string>Min Advance</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="TextBox" name="textBoxSearcherMaxAdvances">
              <property name="text">
               <string notr="true">1000</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="labelSearcherMaxAdvance">
              <property name="text">
               <string>Max Advance</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="TextBox" name="textBoxSearcherMinAdvances">
              <property name="text">
               <string notr="true">0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelSearcherEncounter">
            <property name="text">
             <string>Encounter</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="4">
           <widget class="ComboBox" name="comboBoxSearcherMethod">
            <item>
             <property name="text">
              <string>IVs</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IVs C-Gear</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PIDRNG</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="2" colspan="4">
           <widget class="ComboBox" name="comboBoxSearcherEncounter">
            <item>
             <property name="text">
              <string>Static</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Roamer</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="2" colspan="4">
           <widget class="ComboBox" name="comboBoxSearcherLead">
            <item>
             <property name="text">
              <string>Any</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Synchronize</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cute Charm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="labelSearcherEndDate">
            <property name="text">
             <string>End Date</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="4">
           <widget class="DateEdit" name="dateEditSearcherStartDate"/>
          </item>
          <item row="4" column="2" colspan="4">
           <widget class="DateEdit" name="dateEditSearcherEndDate"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>Filter</class>
   <extends>QWidget</extends>
   <header>Forms/Controls/Filter.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Forms/Controls/ComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header>Forms/Controls/DateEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>tabRNGSelector</tabstop>
  <tabstop>comboBoxGeneratorMethod</tabstop>
  <tabstop>pushButtonGeneratorLead</tabstop>
  <tabstop>comboBoxGeneratorLead</tabstop>
  <tabstop>textBoxGeneratorInitialAdvances</tabstop>
  <tabstop>textBoxGeneratorMaxAdvances</tabstop>
  <tabstop>textBoxGeneratorSeed</tabstop>
  <tabstop>tableViewGenerator</tabstop>
  <tabstop>textBoxSearcherMinAdvances</tabstop>
  <tabstop>textBoxSearcherMaxAdvances</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>tableViewSearcher</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
